### PR TITLE
Added travel reimbursement tests

### PR DESCRIPTION
**Pull request by mcao**

### CODE REVIEW

File: src/assets/schemas/travelReimbursementSchema.json [hunk 1]
```diff
@@ -14,10 +14,10 @@
     },
     "group_members": {
       "enum": [
-        "1",
-        "2",
-        "3",
-        "4+"
+        1,
```
Comment (sushshring) [inline src/assets/schemas/travelReimbursementSchema.json]: This change is incorrect. The DB will reject these values

File: test/models/travel-reimbursement/travel-reimbursement-data-mapper-impl.test.ts [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+import { MysqlUow } from '../../../src/services/database/svc/mysql-uow.service';
+import { Logger } from '../../../src/services/logging/logging';
+
+let travelReimbursemenetDataMapper: TravelReimbursementDataMapperImpl;
+let activeHackathonDataMapper: IActiveHackathonDataMapper;
+let mysqlUow: MysqlUow;
+const mysqlUowMock = mock(MysqlUow);
+const acl: IAcl = new RBAC();
+
+describe('TEST: Event Data Mapper', () => {
```
Comment (sushshring) [inline test/models/travel-reimbursement/travel-reimbursement-data-mapper-impl.test.ts]: Not an event data mapper test

File: src/assets/schemas/travelReimbursementSchema.json (hunk 1 above)
Comment (Demolisty24) [reply src/assets/schemas/travelReimbursementSchema.json]: Hold on this comment

File: src/assets/schemas/travelReimbursementSchema.json [hunk 3]
```diff
@@ -27,4 +27,4 @@
     "mailing_address",
     "group_members"
   ]
-}
+}
```
Comment (sushshring) [inline src/assets/schemas/travelReimbursementSchema.json]: whitespace after. Unnecessary change

File: test/models/travel-reimbursement/travel-reimbursement-data-mapper-impl.test.ts [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+    });
+  });
+
+  describe('TEST: Travel reimbursement delete', () => {
+    // @ts-ignore
+    it('causes the travel reimbursement to get deleted', async () => {
+      // GIVEN: A travel reimbursement with a valid ID to read from
+      const uid = { uid: 'test uid', hackathon: 'test uid' };
+      // WHEN: Retrieving data for this travel reimbursement
+      await travelReimbursemenetDataMapper.delete(uid);
```
Comment (sushshring) [inline test/models/travel-reimbursement/travel-reimbursement-data-mapper-impl.test.ts]: spelling mistake
